test/functional/ex_cmds/verbose_spec.lua

```diff
@@ -0,0 +1,124 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local eq = helpers.eq
+local exec = helpers.exec
+local exec_capture = helpers.exec_capture
+local write_file = helpers.write_file
+local call_viml_function = helpers.meths.call_function
+
+describe('lua verbose:', function()
+  clear{args={'-V1'}}
```

Comment:
this is not in a `before_each` so all of the it() cases below will use the same `nvim` instance. Is that intended? I would at least put a comment here.
```suggestion
  -- All test cases below use the same nvim instance.
  clear{args={'-V1'}}
```

Reply:
Test run faster in same instance . I think tjdevries told me once before in a pr that I didn't need to clear for every test.

Reply:
It should  be in a `setup(function() ... end)` block then otherwise the time when this is run is not well defined and will cause issues soon or later.

Reply:
> It should be in a setup(function() ... end) block then otherwise the time when this is run is not well defined

done .